src/registry.ts

```diff
@@ -185,7 +186,8 @@ export async function getEntry(
       },
     });
   } catch (err) {
-    return handleGetEntryErrResponse(err as AxiosError);
+    // Check the executeRequest error to see if a 404 status was returned.
+    return handleGetEntryErrResponse(err as ExecuteRequestError);
```

Comment:
Should we use `instanceof` and handle both cases?

Reply:
`executeRequest` should only ever throw `ExecuteRequestError` now. I thought about adding something like 

```ts
if (!(err instanceof ExecuteRequestError)) {
  throw err;
}
```

just in case, but I would have to add a test to cover these lines and there's no way to actually trigger them. And an `istanbul-ignore` would be ugly.